docs/DEVELOPING.md

```diff
@@ -343,6 +343,139 @@ $ make ftest NAME=azure_blob_storage
 
+
+## Oracle Database Connector
+
+The [Elastic Oracle connector](https://github.com/elastic/connectors-python/blob/8.7/connectors/sources/oracle.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+⚠️ _Currently, this connector is available in **beta** starting in 8.7_.
+Features in beta are subject to change and are not covered by the service level agreement (SLA) of features that have reached general availability (GA).
+
+Elastic versions 8.6.0+ are compatible with Elastic connector frameworks. Your deployment must include the Elasticsearch, Kibana, and Enterprise Search services.
+
+### Setup and basic usage
+
+Complete the following steps to deploy the connector:
+
+1. [Gather Oracle details](#gather-oracle-details)
+2. [Configure Oracle connector](#configure-oracle-connector)
+
+#### Gather Oracle details
+
+Collect the information that is required to connect to your Oracle:
+
+- The server host address where the Oracle is hosted.
+- The port on which the Oracle server runs
+- The username the connector will use to log in to Oracle.
+- The password the connector will use to log in to Oracle.
+- The Oracle database name to ingest data from.
+
+#### Configure Oracle connector
+
+The following configuration fields need to be provided for setting up the connector:
+
+##### `host`
+
+The server host address where the Oracle server is hosted. Default value is `127.0.0.1`. Examples:
+
+  - `192.158.1.38`
+  - `demo.instance.demo-region.demo.service.com`
+
+##### `port`
+
+The server port where the Oracle server runs on. Default value is `9090`. i.e. For a secured connection, a user needs to configure a SSL port.
+
+  - `5432`
+  - `9090`
+
+##### `user`
+
+The username of the account for Oracle. Default value is `admin`.
+
+##### `password`
+
+The password of the account to be used for the Oracle server. Default value is `Password_123`.
+
+##### `database`
+
+Name of the Oracle database. Default value is `xe`. Examples:
+
+  - `employee_database`
+  - `customer_database`
+
+##### `tables`
+
+Comma-separated list of tables. The Oracle connector will fetch data from all tables present in the configured database, if the value is `*` . Default value is `*`. Examples:
+
+  - `table_1, table_2`
+  - `*`
+
+##### `fetch_size`
+
+Number of rows to fetch on each call. Default value is `50`.
+
+##### `retry_count`
+
+The number of retry attempts after failed call to Oracle. Default value is `3`.
+
+##### `oracle_protocol`
+
+Protocol which the connector uses to establish a connection. Default value is `TCP`. i.e. For a secured connection, a user needs to pass `TCPS`.
+
+##### `oracle_home`
+
+Path of the Oracle home directory to run connector with thick mode for secured connection. Default value is `None`.
```

Comment:
I believe `None` is what is present in the python backend. 

Is this optional?

Reply:
Sure, remove `None` and add Note for empty field